docs/proposal-rose-suite-run.md

```diff
@@ -39,66 +39,228 @@ The remaining points can happen any time after 1) is complete although note that
 ### Key Functionality
 
 1. Installation of suites on platforms and support for moving the share, work &
-   log directories to different locations. Currently this is done by
-   `rose suite-run` but we will include this in the
+   log directories to different locations. This will be included in the
    [platforms support](proposal-platforms.md#enhancements-to-support-rose-suite-run-functionality).
 
-2. On start up and reload, install the suite from the source location (working
-   copy) to the run location (`cylc-run`).
-   * Currently this functionality assumes a shared filesystem but we should
-     support workflow server hosts with separate filesystems (future
-     enhancement?).
-   * Needs to create the run time directory structure which has been configured
-     for the `localhost` platform.
+2. Support installation of a suite from the source location (working copy) to
+   the run location (`cylc-run`).
+   * This functionality assumes the source location (working copy) and the run
+     location (`cylc-run`) are both accessible on the host where the command is
+     run.
```

Comment:
Reinforced by the creation of a `source` symlink in the installed workflow (as presently generated during registration) coupling the installation and source.